src/view/View.js

```diff
@@ -1070,7 +1072,7 @@ new function() { // Injection scope for event handling on the browser
     // Touch handling inspired by Hammer.js
     var navigator = window.navigator,
         mousedown, mousemove, mouseup;
-    if (navigator.pointerEnabled || navigator.msPointerEnabled) {
+    if (window.PointerEvent || navigator.msPointerEnabled) {
```

Comment:
I think it should now be: `window.PointerEvent || navigator.MSPointerEvent`, see: https://github.com/hammerjs/hammer.js/blob/563b5b1e4bfbb5796798dd286cd57b7c56f1eb9e/src/input/pointerevent.js#L34